examples/RNN-examples.ipynb

```diff
@@ -2,7 +2,16 @@
  "cells": [
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
```

Comment:
Please do not save it, because it will create a lot of diffs in the future. It is not idempotent

Reply:
Yes I know it's a lot of diffs, do you know how to avoid that ?

Reply:
If I needed to fix some issues in a notebook, I run it manually, then I committed and after that check using papermill if it works. Worth to check https://papermill.readthedocs.io/en/latest/usage-cli.html. Particularly `--no-request-save-on-cell-execute`. IMO we use papermill for nonstandard case :)